src/app/qgssettingstree.cpp

```diff
@@ -319,7 +321,17 @@ QTreeWidgetItem *QgsSettingsTree::createItem( const QString &text,
     item->setFlags( item->flags() | Qt::ItemIsEditable );
 
   item->setData( 0, TypeRole, isGroup ? Group : Setting );
-  item->setData( 0, PathRole, mSettings->group().isEmpty() ? text : mSettings->group() + '/' + text );
+
+  QString completeSettingsPath = mSettings->group().isEmpty() ? text : mSettings->group() + '/' + text;
+  item->setData( 0, PathRole, completeSettingsPath );
+
+  // If settings registered add description
+  if ( !isGroup )
+  {
+    const QgsSettingsEntryBase *settingsEntry = QgsApplication::settingsRegistryCore()->getSettingsEntry( completeSettingsPath, true );
+    if ( settingsEntry != nullptr )
```

Comment:
```suggestion
    if ( ! settingsEntry )
```

Reply:
thank you, just without the ! i guess :)

Reply:
oh, yes.